scripts/publish.js

```diff
@@ -81,6 +81,7 @@ const createStarterDist = async (basename, isTypescript = false) => {
 
   // copy root files
   const rootFiles = [
+    ".nvmrc",
```

Comment:
Add the `.nvmrc` to the files from the root project we copy over in the creation of each "flavor" of the starter.